kuksa-persistence-provider/Readme.md

```diff
@@ -0,0 +1,71 @@
+# KUKSA Persistence Provider
+
+All data in KUKSA is ephemereal. However, in a car there is often data that does not change over the lifetime of a vehicle, and data where you want changes to be persisted over ignition cycles.
+
+This provider can achieve this. It can restore certain values upon startup, either sensor (current) values, or actuations.
```

Comment:
Have you put soe thought into how this would work for actuations with the new concept that will be introduced in the kuksa.val.v2?

Reply:
Not that good I think - Isn't the thoughts in kuksa.val.v2 that there should only be a single instance interested in the target/actuation-value? I.e. there cannot be multiple entities interested in the same actuation value?